### PR TITLE
Go 1.21.8 1.22.1

### PR DESCRIPTION
**Pull request by Fabio1988**
I am still not sure how related those two repos are :smile: 
What I understood this repo contains the images.

https://github.com/crazy-max/xgo/pull/123

### CODE REVIEW

**Comment by crazy-max** [comment]
> I am still not sure how related those two repos are 😄

xgo depends on goxx so needs first goxx update before creating a PR on xgo

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
     needs: prepare
     runs-on: ubuntu-latest
     env:
-      LATEST_CURRENT: 1.22.0
+      LATEST_CURRENT: 1.22.1
       LATEST_PREVIOUS: 1.21.7
```
[the remark this answers is not in the document]
Comment (Fabio1988) [reply .github/workflows/build.yml]: sorry forgot about it, done :)

**Comment by Fabio1988** [comment]
> See my comment and also squash your commits, thanks!

Thanks a lot, squashed and fixed :)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -71,6 +71,7 @@ jobs:
           - 1.21.6
           - 1.21.7
```
Comment (crazy-max) [inline .github/workflows/build.yml:72]: missing bump here

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ jobs:
       fail-fast: false
       matrix:
         go_version:
-          - 1.22.0
+          - 1.22.1
           - 1.21.7
```
Comment (crazy-max) [inline .github/workflows/test.yml:25]: here as well

**Review by crazy-max — CHANGES_REQUESTED** [review]
check other places where this should be bumped to 1.21.8 similar to https://github.com/crazy-max/goxx/pull/68, thx

**Review by crazy-max — CHANGES_REQUESTED** [review]
Align commit message as well to `Go 1.21.8 1.22.1`

**Comment by Fabio1988** [comment]
> Align commit message as well to `Go 1.21.8 1.22.1`

same thought at same time,done :smile:

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**